BootloaderCorePkg/Library/SmbiosInitLib/SmbiosInitLib.c

```diff
@@ -18,6 +18,32 @@
 
 VOID      *mType127Ptr            =   NULL;
 
+//
+// Add more platform specific strings
+// in the following format
+//
+CONST SMBIOS_TYPE_STRINGS  mDefaultSmbiosStrings[] = {
+  // Type                               StrId   String
+  // Type 0
+  {  SMBIOS_TYPE_BIOS_INFORMATION ,        1,  "Intel Corporation"    },  // Vendor
+  {  SMBIOS_TYPE_BIOS_INFORMATION ,        2,  "SB_XXXX.XXX.XXX.XXX"  },  // BIOS Version
+  {  SMBIOS_TYPE_BIOS_INFORMATION ,        3,  "Unknown Date"         },  // BIOS Release Date
+  // Type 1
+  {  SMBIOS_TYPE_SYSTEM_INFORMATION ,      1,  "Intel Corporation"    },  // Manufacturer
+  {  SMBIOS_TYPE_SYSTEM_INFORMATION ,      2,  "Unknown Platform"     },  // Product Name
+  {  SMBIOS_TYPE_SYSTEM_INFORMATION ,      3,  "Platform ver-XXX.XXX.XXXXX.X" },  // Version
+  {  SMBIOS_TYPE_SYSTEM_INFORMATION ,      4,  "Unknown Serial no",   },  // Serial Number
+  {  SMBIOS_TYPE_SYSTEM_INFORMATION ,      5,  "Unknown SKU"          },  // SKU Variant
+  {  SMBIOS_TYPE_SYSTEM_INFORMATION ,      6,  "Client Platform",     },  // Serial Number
+  // Type 2
+  {  SMBIOS_TYPE_BASEBOARD_INFORMATION ,   1,  "Unknown Serial no"    },  // Manufacturer
+  {  SMBIOS_TYPE_BASEBOARD_INFORMATION ,   2,  "Unknown Product"      },  // Product Name
+  {  SMBIOS_TYPE_BASEBOARD_INFORMATION ,   3,  "Build ver X-XXXXXXXXXXXXXXXXXX" },  // Version
+  {  SMBIOS_TYPE_BASEBOARD_INFORMATION ,   4,  "Unknown ProdNumber",  },  // Serial Number
+  // Type 127 - End of strings
+  {   SMBIOS_TYPE_END_OF_TABLE,            0,  "End Of String Array"  },
+};

```

Comment:
Please use #define to save data section size since most of strings are "Unknown".
```
#define SMBIOS_DEF_VENDOR "Intel"
#define SMBIOS_UNKNOWN "Unknown"
```
And, "End Of String Array" is not necessary. Just "" would be enough.